app/controllers/concerns/sufia/generic_works_controller_behavior.rb

```diff
@@ -15,6 +15,7 @@ module GenericWorksControllerBehavior
     def new
       # TODO: move this to curation_concerns
       @form = form_class.new(curation_concern, current_ability)
+      curation_concern.apply_depositor_metadata(current_user)
```

Comment:
Will do.  I originally had it set like that, but only found `depositor=` used inside of `apply_depositor_metadata` so I thought I might have been missing other important attributes.